### DISCUSSION TITLE
馬斯克首次與推特員工面對面！掛保證會完成收購，評估是否遠距辦公

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#446] [General]
馬斯克首次與推特員工面對面！掛保證會完成收購，評估是否遠距辦公
馬斯克真的是一個狂人哈哈哈
https://tw.news.yahoo.com/%E9%A6%AC%E6%96%AF%E5%85%8B%E9%A6%96%E6%AC%A1%E8%88%87%E6%8E%A8%E7%89%B9%E5%93%A1%E5%B7%A5%E9%9D%A2%E5%B0%8D%E9%9D%A2-%E6%8E%9B%E4%BF%9D%E8%AD%89%E6%9C%83%E5%AE%8C%E6%88%90%E6%94%B6%E8%B3%BC-%E8%A9%95%E4%BC%B0%E6%98%AF%E5%90%A6%E9%81%A0%E8%B7%9D%E8%BE%A6%E5%85%AC-005120142.html

**Comment by BingRong1217** [comment]
馬斯克近年來是大名人呢！

**Comment by limitx0** [comment]
馬斯克真的是把推特搞得人仰馬翻
不知道之後會怎樣就是了
